www.foia.gov/_layouts/default.html

```diff
@@ -0,0 +1,23 @@
+---
+---
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```

Comment:
:+1: that makes sense, do you mind if we punt until we know more about the new template will contain?